sdk/postgresql/Azure.ResourceManager.PostgreSql/src/PostgreSqlFlexibleServers/Custom/Models/PostgreSqlBaseCapability.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
```

Comment:
we always disable nullable in SDK code
```suggestion
#nullable disable
```